test/hack/cleanup/main.go

```diff
@@ -18,20 +18,26 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"os/exec"
+	"strings"
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
-	"github.com/aws/aws-sdk-go-v2/service/iam"
-	"github.com/samber/lo"
-	"go.uber.org/zap"
+	"github.com/aws/aws-sdk-go-v2/service/eks"
 
 	"github.com/aws/karpenter/test/hack/cleanup/metrics"
 	"github.com/aws/karpenter/test/hack/cleanup/resourcetypes"
+
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/samber/lo"
+	"go.uber.org/zap"
 )
 
 const expirationTTL = time.Hour * 12
+const soakExpirationTTL = time.Hour * 168 // 7 Days
```

Comment:
AFAIK, everything should be the same in terms of resource cleanup except for the tagging that's associated with the cluster resources. Can we just make this code work as-is; the only difference being that we have a tag like `testing/type=soak`